policyengine_us/parameters/gov/states/la/tax/income/deductions/standard/amount.yaml

```diff
@@ -0,0 +1,24 @@
+description: Louisiana provides the following standard deduction amount, depending on filing status. 
```

Comment:
the louisiana gov. limits the itemized deduction amount to the federal standard deduction amount, which is already in the model.